tests/unit/sokoban_test.cpp

```diff
@@ -50,4 +50,581 @@ TEST_CASE("should not be solved when empty goal cell is present") {
     }};
     Sokoban soko(levels);
     CHECK(!soko.solved());
-}
\ No newline at end of file
+}
+
+/************** BEGIN TESTS FOR MOVE (PLAYER ONLY) **************/
```

Comment:
I'd try https://github.com/doctest/doctest/blob/master/doc/markdown/testcases.md#test-suites instead of these comments